cli/src/commands/mod.rs

```diff
@@ -94,7 +93,8 @@ enum Command {
     Edit(edit::EditArgs),
     #[command(subcommand)]
     File(file::FileCommand),
-    Files(files::FilesArgs),
+    #[command(hide = true)]
```

Comment:
Nit: Needs a deprecation comment and should not be hidden yet.

Reply:
all the other deprecated things I see here are marked hidden.  Not sure what the pattern to follow is.

Reply:
I am dumb, you can keep it as is.